samples/CosmosDbIntegtation/WriteAroundSamples/StreamSample.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+using StackExchange.Redis;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Redis.Samples
+{
+    internal class StreamSample
+    {
+        // Redis database and stream stored in local.settings.json
+        private static readonly IDatabase redisDB = ConnectionMultiplexer.Connect(Environment.GetEnvironmentVariable("redisConnectionString")).GetDatabase();

```

Comment:
You can attempt to do a Lazy load of this if you're interested in learning about it https://learn.microsoft.com/en-us/dotnet/framework/performance/lazy-initialization